expressEngine/product/service/product.js

```diff
@@ -2,29 +2,23 @@
 import { BaseService, DBService } from '../../lib/service/index';
 
 // import collections
-import { ProductModel, ProductDetail } from '../../model/index';
-
-// import messages
-import {success} from '../../cms/product/index';
-
+import { product } from '../../model/index';
```

Comment:
I don't think there is a need of including the index in the import statement

Reply:
Done.